tests/phpunit/src/CommandTestBase.php

```diff
@@ -228,13 +228,45 @@ public function mockAcsfEnvironmentsRequest(
     return $response;
   }
 
+  /**
+   * @param object $applications_response
+   *
+   * @return object
+   * @throws \Psr\Cache\InvalidArgumentException
+   */
+  public function mockCloudEnvironmentsRequest(
```

Comment:
Apparently we were only mocking ACSF requests, leading to a lot of uncovered code paths especially in the push/pull commands (wherever `isAcsf()` was checked). This should make it easier to test those code paths (we'll still need to write new test cases for them).